### ISSUE TITLE
'charmap' codec can't decode byte 0x9d in position 180: character maps to <undefined>

### ISSUE THREAD

**Issue by harshitpoornima1008** [FintelLabs/emosent-py#2]
I am getting the above stated error kindly help me to solve this

'charmap' codec can't decode byte 0x9d in position 180: character maps to <undefined>

I think it is unable to decode the csv file

**Comment by om-hb** [comment]
Look at pull request #1, it will most likely solve this problem for you!

**Comment by omkar-foss** [comment]
Thanks for pointing this out @om-hb :+1: 

@harshitpoornima1008 This issue that you mentioned above has been resolved in #1 and has been merged to `master`. I'll make a new release of the package on PyPI soon. 

Meanwhile you can install the new (fixed) version directly from GitHub as follows:
```
python -m pip install https://github.com/FintelLabs/emosent-py/archive/e5788e73e88f691b44fd5f6eda72187b559d99f2.tar.gz
```

**Comment by omkar-foss** [comment]
**This above mentioned issue has been fixed in [release v0.1.7](https://github.com/omkar-foss/emosent-py/releases/tag/v0.1.7).**

You can install in via `pip` as follows:

```python
pip install emosent-py==0.1.7
```

My sincere apologies on the delayed response, I've been inactive for quite a long time due to some personal reasons. But I'm trying my best to actively maintain this project going forward. Please feel free to reopen this issue in case you still face the issue using the new release (v0.1.7). Thanks.
